trifid/values.yaml

```diff
@@ -14,6 +7,35 @@ trifid:
       password: pass
       # secretName: # name of an existing secret
 
+replicaCount: 1
+
+image:
+  repository: docker.io/zazuko/trifid
+  tag: 2.3.2
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
```

Comment:
Same here, let's make these the default (i.e. uncomment, except `readOnlyRootFilesystem`)